addons/sale_management/models/sale_order.py

```diff
@@ -162,9 +181,12 @@ def _compute_name(self):
             if line.product_id and line.order_id.sale_order_template_id:
                 for template_line in line.order_id.sale_order_template_id.sale_order_template_line_ids:
                     if line.product_id == template_line.product_id:
-                        line.name = template_line.with_context(lang=line.order_id.partner_id.lang).name + line._get_sale_order_line_multiline_description_variants()
+                        line.name = template_line.with_context(lang=line.order_id.partner_id.lang).name + line._get_sale_order_line_description_from_template()
```

Comment:
The call to `_get_sale_order_line_description_from_template` is not done with the partner lang in the context.